state-of-our-networks.md

```diff
@@ -1,11 +1,87 @@
+<div class="event-time-location">
+  <div class="event-meta">
+    <i class="icon fa fa-ticket" aria-hidden="true"></i>
+    <p class="event-cost">$10 suggested donation, <a href="#registration">register below</a>
+    </p>
+  </div>
+  <div class="event-meta">
+    <i class="icon fa fa-calendar-o" aria-hidden="true"></i>
+    <p class="event-time">{{ page.date | date: "%-d %B %Y"}}, <strong>{{page.startTime | date:"%-I:%M%P "}}</strong> – <strong>{{page.endTime | date:"%-I:%M%P "}}</strong></p>
+  </div>
+  <div class="event-meta">
+    <i class="icon fa fa-map-marker" aria-hidden="true"></i>
+    <p class="event-location">
+    {% if page.locationLink %}
+      <a href="{{page.locationLink}}" target="_blank">{{ page.location }}</a> <!--_-->
+    {% else %}
+      {{ page.location }}
+    {% endif %}
+    </p>
+  </div>
+</div>
 
 ***
 
-More information and registration details to follow. In the meantime, come say hi on our chat, or email us at [{{ site.email }}](mailto:{{ site.email }}), to find out more!
+Brought to you by Toronto Mesh, '**State of our Networks**' is a one-day, hands-on conference about the past, present, and future of building our own network infrastructures to communicate and access the internet in our city.
+
+On {{ page.date | date: '%B %d, %Y' }} from {{page.startTime | date:"%-I:%M%P "}} to {{page.endTime | date:"%-I:%M%P "}} join us for a one-day hands-on conference to:
+
+- learn and participate in hands on workshops led by current networking projects in the city of Toronto
+- hear about the history of community networks in our city
+- discuss how we can build a resilient network that addresses barriers in the future
+
+Toronto Mesh welcomes people from all backgrounds and technical skill levels and aims to provide a harassment-free environment. This event has a [Code of Conduct](https://tomesh.net/code-of-conduct/).
+
+This page will be updated with [full schedule](#schedule) closer to the event date.
+
+## Registration
+
+Register by email, **$10** suggested donation at the door, no one turned away for lack of funds.
+
+<div class="">
+  <a class="button" href="mailto:{{ site.email }}?subject=State of Our Networks Registration" role="link"><i class="fa fa-ticket" aria-hidden="true"></i>&nbsp;&nbsp;Register Now</a>
+</div>
+
+<br />
+
+## Location
+
+Event and registration will be in the **Semaphore Demo Room, BL 417** on the fourth floor of the {% if page.locationLink %} <a href="{{page.locationLink}}" target="_blank">Claude T. Bissell building</a><!--_-->{% else %} Claude T. Bissell building{% endif %}.
+
+The closest TTC Subway stop is St. George (at Bloor Street and St. George Street) and Streetcar stop is Spadina Ave at Sussex Street.
+
+<iframe width='100%' height='250px' frameBorder='0' src='https://a.tiles.mapbox.com/v4/dcwalk.27eme78b/attribution,zoompan,zoomwheel.html?access_token=pk.eyJ1IjoiZGN3YWxrIiwiYSI6ImNpZ3NzaWljdzA1ajdzeGtudTNzM3NnanYifQ.kA_-f8oD-sPwjPXfqcv1og'></iframe>
```

Comment:
The marker looks to be off-center. Also, I think the zoom level can be one level in so that street names are visible along with disabling mouse wheel scroll as page scroll gets caught in the iframe.